syncd/Workaround.cpp

```diff
@@ -70,6 +71,15 @@ bool Workaround::isSetAttributeWorkaround(
         return true;
     }
 
+    if (doingComparisonLogic) {
+        if (objectType == SAI_OBJECT_TYPE_TUNNEL && attrId == SAI_TUNNEL_ATTR_ENCAP_TTL_MODE) {
+            SWSS_LOG_WARN("setting %s failed: %s, not all platforms support this attribute when doing comparison logic of 202411->202505 warm upgrade",
```

Comment:
Should you be checking for if this upgrade is cross branch? Your logic will prevent these SET operations irrespective of the upgrade path. But your log here assumes a different intention.